client/my-sites/people/people-invite-details/index.jsx

```diff
@@ -19,17 +19,33 @@ import HeaderCake from 'components/header-cake';
 import Card from 'components/card';
 import PeopleListItem from 'my-sites/people/people-list-item';
 import Gravatar from 'components/gravatar';
+import Button from 'components/button';
 import QuerySiteInvites from 'components/data/query-site-invites';
 import EmptyContent from 'components/empty-content';
 import { getSelectedSite } from 'state/ui/selectors';
-import { isRequestingInvitesForSite, getInviteForSite } from 'state/invites/selectors';
+import {
+	isRequestingInvitesForSite,
+	getInviteForSite,
+	isDeleting,
+	didDeletionSucceed,
+} from 'state/invites/selectors';
+import { deleteInvite } from 'state/invites/actions';
 
 class PeopleInviteDetails extends React.PureComponent {
 	static propTypes = {
 		site: PropTypes.object,
 		inviteKey: PropTypes.string.isRequired,
 	};
 
+	componentWillReceiveProps( nextProps ) {
+		if (
+			true === nextProps.deleteSuccess &&
+			nextProps.deleteSuccess !== this.props.deleteSuccess
```

Comment:
These two conditionals can be simplified to `nextProps.deleteSuccess && ! this.props.deleteSuccess`.